packages/create-youtrack-app/_templates/init/vite-app/@types/globals.d.ts.t

```diff
@@ -62,7 +60,7 @@ export interface EmbeddableWidgetAPI extends PluginEndpointAPILayer {
 
   loadServices: (applicationName: string) => Promise<HubService[]>;
 
-  alert: (...args: Parameters<(typeof AlertService)['addAlert']>) => Promise<void>;
```

Comment:
Previously the return value of this method was `Promise<void>`, now it is `void`. Is it OK?